### PR TITLE
Create codeql.yml & license

### PR DESCRIPTION
**Pull request by cmwilson21**
### Why:

Enabling CodeQL for the repo

### What's being changed:

Added a CodeQL file for the game

### CODE REVIEW

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
+  schedule:
+    - cron: '33 7 * * 3'
```
Comment (heiskr) [inline .github/workflows/codeql.yml:21]: ?

**Comment by Lambok134** [comment]
Ok

**Comment by Lambok134** [comment]
Aprov

**Comment by Lambok134** [comment]
Ik

**Pull request closed without merging**